utils/build.ps1

```diff
@@ -1801,7 +1819,17 @@ if ($Stage) {
   Stage-BuildArtifacts $HostArch
 }
 
-if ($Test -contains "swift") { Build-Compilers $HostArch -Test }
+if ($Test -ne $null -and (Compare-Object $Test @("clang", "lld", "lldb", "llvm", "swift") -PassThru -IncludeEqual -ExcludeDifferent) -ne $null) {
+  $Tests = @{
+    "-TestClang" = $Test -contains "clang";
+    "-TestLLD" = $Test -contains "lld";
+    "-TestLLDB" = $Test -contains "lldb";
+    "-TestLLVM" = $Test -contains "llvm";
+    "-TestSwift" = $Test -contains "swift";
+  }
+  Build-Compilers $HostArch @Tests
```

Comment:
You could avoid splatting by using `-TestClang:($Test -contains "clang")` (`-Foo:$true` / `-Foo:$false` is valid syntax)

Reply:
Yeah, that is certainly a viable option as well.  I find it difficult to tell which is better.  The collection as is does make it easier to tell at a glance what options are being setup.  I would defer to your expertise if one is better than the other.

Reply:
I'm going to merge this for now, but happy to revisit if you have suggestions here.